docs/solutions/connect_vcns_using_multiple_vnics/output.tf

```diff
@@ -0,0 +1,28 @@
+# Outputing required info for users
+output "Bridge Instance Public IP" {
+    value = "${data.oci_core_vnic.BridgeInstanceVnic1.public_ip_address}"
+}
+
+output "PrivateInstance1 Private IP" {
+    value = "${oci_core_instance.PrivateInstance.private_ip}"
+}
+
+output "PrivateInstance2 Private IP" {
+    value = "${oci_core_instance.PrivateInstance2.private_ip}"
+}
+
+output "SSH login to the Bridge Instance" {
+    value = "ssh -A opc@${data.oci_core_vnic.BridgeInstanceVnic1.public_ip_address}"
```

Comment:
These instructions in the output are great. Do we want to use -A here though? I was able to login into instance 1 with the following, after setting the variables in the command:
ssh -i $PRIVATE_KEY_PATH -o ProxyCommand="ssh -i $PRIVATE_KEY_PATH opc@$BASTION_IP -W %h:%p %r" opc@$PRIVATE_INSTANCE_IP

Reply:
From bridge instance, they have to login to the private instance for which forwarding with -A is required.